docs/content/config/setup.sh.md

```diff
@@ -66,7 +66,7 @@ OPTIONS
 
         -p PATH
             Provides the config folder path. The default is
-            /home/maxwell/Dokumente/github/docker-mailserver/config/
+            /home/maxwell/Documents/github/docker-mailserver/config/
```

Comment:
Is this true? Does `-p` have a default value and why is it prefixed with maxwell 😆

Reply:
Because I copied it from a VM with a user called `maxwell` 😂 I will change that...